packages/wrangler/src/dev/local.tsx

```diff
@@ -59,6 +59,18 @@ function useLocalWorker({
   const local = useRef<ReturnType<typeof spawn>>();
   const removeSignalExitListener = useRef<() => void>();
   const [inspectorUrl, setInspectorUrl] = useState<string | undefined>();
+  // if we're using local persistence for data, we should use the cwd
+  // as an explicit path, or else it'll use the temp dir
+  // which disappears when dev ends
+  const localPersistencePathOrDisableLocalPersistence = enableLocalPersistence
+    ? // Maybe we could make the path configurable as well?
```

Comment:
Configurable via the TOML or a flag?

Reply:
I like the way miniflare handles things, sure could be under another key to not expose the "miniflare" name, but the way it's handled there (default or specific path per KV, DO and cache) is pretty neat. 

A flag is also important IMO due to integration test ci/cd scenarios, where loading a test data set is important, and dynamic cache folder paths quickly become cumbersome with wrangler.toml